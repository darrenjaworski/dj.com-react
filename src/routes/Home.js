import React from "react";

import { Heading } from "../components/heading";
import Paragraph from "../components/paragraph";
import { TextLink } from "../components/links";

const Home = props => {
  return (
    <>
      <Heading>home</Heading>
      <Paragraph>
        Welcome. This is my home on the web. I dramatically simplified the site
        and I hope that you like it. I'm always available for public comment.
        Please send an email to{" "}
        <TextLink href="mailto:darrenjaworski@gmail.com">
          darrenjaworski@gmail.com
        </TextLink>
        {". "}
        (Please allow 10-15 months for response.)
      </Paragraph>
      <Paragraph>
        Check out my past work in{" "}
        <TextLink link="/journalism">journalism</TextLink> and what I'm
        currently hacking away at in{" "}
        <TextLink link="/projects">projects</TextLink>.
      </Paragraph>
      <Paragraph>
        As always you can find me on{" "}
        <TextLink href="https://twitter.com/darrenjaws">Twitter</TextLink>,{" "}
        <TextLink href="https://www.facebook.com/DarrenJJaworski">
          Facebook
        </TextLink>
        ,{" "}
        <TextLink href="https://www.instagram.com/darrenjaws/">
          Instagram
        </TextLink>
        ,{" "}
        <TextLink href="https://www.linkedin.com/in/darrenjaworski">
          LinkedIn
        </TextLink>
        , <TextLink href="https://github.com/darrenjaworski">Github</TextLink>,{" "}
        <TextLink href="http://bl.ocks.org/darrenjaworski">
          bl.ocks.org
        </TextLink>
        , 
        <TextLink href="https://medium.com/@DarrenJaws">Medium</TextLink>,{" "}
        <TextLink href="https://www.strava.com/athletes/3266824">
          Strava
        </TextLink>
        ... (I'm sure I'm missing others. You get the point. Troll away.)
      </Paragraph>
      <Paragraph>
        For those interested in watching me wear a tie and answer questions:{" "}
        <TextLink href="https://docs.google.com/document/d/19L1W3PXUyboaUWB0shDedjKwzRPsqiBw0VxsAir45EU/edit?usp=sharing">
          résumé
        </TextLink>
        .
      </Paragraph>
      <Paragraph>Now with dark theme.</Paragraph>
    </>
  );
};

export default Home;
