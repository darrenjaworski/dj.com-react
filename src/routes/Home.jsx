import React from "react";

import { Heading } from "../components/heading";
import Paragraph from "../components/paragraph";
import { TextLink } from "../components/links";

const Home = () => (
  <>
    <Heading>home</Heading>
    <Paragraph>
      Welcome. This is my home on the web. I dramatically simplified the site
      and I hope that you like it. I&apos;m always available for public comment.
      Please send an email to&nbsp;
      <TextLink href="mailto:darrenjaworski@gmail.com">
        darrenjaworski@gmail.com
      </TextLink>
      . (Please allow 10-15 months for response.)
    </Paragraph>
    <Paragraph>
      Check out my past work in&nbsp;
      <TextLink link="/journalism">journalism</TextLink>
      &nbsp;and what I&apos;m currently hacking away at in&nbsp;
      <TextLink link="/projects">projects</TextLink>.
    </Paragraph>
    <Paragraph>
      As always you can find me on&nbsp;
      <TextLink href="https://twitter.com/darrenjaws">Twitter</TextLink>
      {", "}
      <TextLink href="https://www.facebook.com/DarrenJJaworski">
        Facebook
      </TextLink>
      {", "}
      <TextLink href="https://www.instagram.com/darrenjaws/">
        Instagram
      </TextLink>
      {", "}
      <TextLink href="https://www.linkedin.com/in/darrenjaworski">
        LinkedIn
      </TextLink>
      {", "}
      <TextLink href="https://github.com/darrenjaworski">Github</TextLink>
      {", "}
      <TextLink href="https://medium.com/@DarrenJaws">Medium</TextLink>
      {", "}
      <TextLink href="https://www.strava.com/athletes/3266824">
        Strava
      </TextLink>{" "}
      ... (I&apos;m sure I&apos;m missing others. You get the point. Troll
      away.)
    </Paragraph>
    <Paragraph>
      For those interested in watching me wear a tie and answer questions:&nbsp;
      <TextLink href="https://docs.google.com/document/d/19L1W3PXUyboaUWB0shDedjKwzRPsqiBw0VxsAir45EU/edit?usp=sharing">
        résumé
      </TextLink>
      .
    </Paragraph>
    <Paragraph>Now with dark theme.</Paragraph>
  </>
);

export default Home;
