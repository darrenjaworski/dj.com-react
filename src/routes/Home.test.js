import React from "react";
import renderer from "react-test-renderer";

import { BrowserRouter as Router } from "react-router-dom";
import Home from "./Home";

describe("Home", () => {
  it("renders without crashing", () => {
    const tree = renderer
      .create(
        <Router>
          <Home />
        </Router>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
