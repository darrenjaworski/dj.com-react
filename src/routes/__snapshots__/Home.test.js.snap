// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders without crashing 1`] = `
Array [
  <h1
    className="css-lozscu"
  >
    home
  </h1>,
  <p
    className="css-z4ydkm"
  >
    Welcome. This is my home on the web. I dramatically simplified the site and I hope that you like it. I'm always available for public comment. Please send an email to
     
    <a
      className="css-alkraz"
      href="mailto:darrenjaworski@gmail.com"
    >
      darrenjaworski@gmail.com
    </a>
    . 
    (Please allow 10-15 months for response.)
  </p>,
  <p
    className="css-z4ydkm"
  >
    Check out my past work in
     
    <a
      href="/journalism"
      onClick={[Function]}
    >
      <a
        className="css-alkraz"
      >
        journalism
      </a>
    </a>
     and what I'm currently hacking away at in
     
    <a
      href="/projects"
      onClick={[Function]}
    >
      <a
        className="css-alkraz"
      >
        projects
      </a>
    </a>
    .
  </p>,
  <p
    className="css-z4ydkm"
  >
    As always you can find me on
     
    <a
      className="css-alkraz"
      href="https://twitter.com/darrenjaws"
    >
      Twitter
    </a>
    ,
     
    <a
      className="css-alkraz"
      href="https://www.facebook.com/DarrenJJaworski"
    >
      Facebook
    </a>
    ,
     
    <a
      className="css-alkraz"
      href="https://www.instagram.com/darrenjaws/"
    >
      Instagram
    </a>
    ,
     
    <a
      className="css-alkraz"
      href="https://www.linkedin.com/in/darrenjaworski"
    >
      LinkedIn
    </a>
    , 
    <a
      className="css-alkraz"
      href="https://github.com/darrenjaworski"
    >
      Github
    </a>
    ,
     
    <a
      className="css-alkraz"
      href="http://bl.ocks.org/darrenjaworski"
    >
      bl.ocks.org
    </a>
    , 
    <a
      className="css-alkraz"
      href="https://medium.com/@DarrenJaws"
    >
      Medium
    </a>
    ,
     
    <a
      className="css-alkraz"
      href="https://www.strava.com/athletes/3266824"
    >
      Strava
    </a>
    ... (I'm sure I'm missing others. You get the point. Troll away.)
  </p>,
  <p
    className="css-z4ydkm"
  >
    For those interested in watching me wear a tie and answer questions:
     
    <a
      className="css-alkraz"
      href="https://docs.google.com/document/d/19L1W3PXUyboaUWB0shDedjKwzRPsqiBw0VxsAir45EU/edit?usp=sharing"
    >
      résumé
    </a>
    .
  </p>,
  <p
    className="css-z4ydkm"
  >
    Now with dark theme.
  </p>,
]
`;
